js/lib/variables.css

```diff
@@ -14,7 +14,7 @@
  * so the formatting should be consistent (i.e. no '}' inside a rule).
  */
 
-:root {
+:root .cm-s-light {
```

Comment:
does this scoping affect the rest of the editor styling? If so, would it make sense to use a more generic classname like `devtools-light-theme`? Should we be using [variables.css](https://dxr.mozilla.org/mozilla-central/source/devtools/client/themes/variables.css) directly?